modules/debug_cmd/debug_cmd.c

```diff
@@ -253,6 +253,28 @@ static int cmd_sip_debug(struct re_printf *pf, void *unused)
 	return sip_debug(pf, uag_sip());
 }
 
+static int cmd_sip_trace(struct re_printf *pf, void *arg)
+{
+	struct cmd_arg *carg = arg;
+	const char *prm = carg->prm;
+
+	/* assume false since the trace state is not exposed from libre */
+	static bool enabled = false;
+
+	if (prm) {
+		enabled = !str_casecmp(prm, "on");
```

Comment:
(Sorry, just saw): You can use `str_bool(&enabled, prm)` here. so `true, false...` works too.